diff --git a/Smoothie/SmoothieIngredients.cs b/Smoothie/SmoothieIngredients.cs
index 713bf83..d1551b9 100644
--- a/Smoothie/SmoothieIngredients.cs
+++ b/Smoothie/SmoothieIngredients.cs
@@ -125,18 +125,6 @@ namespace KitchenDrinksMod.Smoothie
                 UnblendedModelChild = "Ingredients/Nuts",
             },
             new SmoothieIngredient
-            {
-                Name = "mandarins",
-                BaseItem = () => Refs.Find<Item>(ItemReferences.MandarinSlice),
-                Instructions = "one mandarin slice",
-                CardNumber = 1,
-                StartingIngredients = new() { () => Refs.Find<Item>(ItemReferences.MandarinRaw) },
-                Color = MakeColor(240, 129, 19, 255),
-                ColorblindLabel = "Ma",
-                IconString = MakeIcon("orange"),
-                UnblendedModelChild = "Ingredients/Mandarins",
-            },
-            new SmoothieIngredient
             {
                 Name = "lemons",
                 BaseItem = () => Refs.Find<Item>(ItemReferences.LemonSliced),
